go/plugins/googlegenai/models.go

```diff
@@ -58,6 +58,7 @@ var (
 				"gemini-1.5-flash-002",
 			},
 			Supports: &gemini.Multimodal,
+			Stage:    ai.ModelStageStable,
```

Comment:
A more accurate way to determine the current stage of a model would be to set the stage value for every version available in the model.
https://cloud.google.com/vertex-ai/generative-ai/docs/learn/model-versions#stable-version